cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -400,6 +400,7 @@ fn in_int_reg(ty: ir::Type) -> bool {
     match ty {
         types::I8 | types::I16 | types::I32 | types::I64 => true,
         types::B1 | types::B8 | types::B16 | types::B32 | types::B64 => true,
+        types::R32 | types::R64 => true,
```

Comment:
Here and below, it concerns me that we are accepting `R32` on a 64-bit target without comment.  IIUC, if anything claiming to be `R32` turns up on a 64-bit target, we have gone way off the rails -- perhaps even to the extent of having a security problem.  In such a circumstance I would much prefer that we bring the system to a halt, even in release builds (== panic in every such `match` on `ty`s); and similarly for x64 stubs, if you have similarly modified them.

Reply:
Done, excellent point.